### PR TITLE
move backend config schemas to where they belong

### LINKED ISSUE #26160: Add missing config schema from backend-common to backend-defaults

### PR DESCRIPTION
**Pull request by freben**
Fixes #26160

### CODE REVIEW

**Review by benjdlambert — APPROVED** [review]

File: packages/backend-common/config.d.ts [hunk 1]
```diff
@@ -221,6 +224,16 @@ export interface Config {
       optionsSuccessStatus?: number;
     };
 
+    /**
+     * Content Security Policy options.
```
Comment (freben) [inline packages/backend-common/config.d.ts:228]: sorry, unrelated change, just made order consistent with the backend-defaults file
